src/components/atoms/video-tile.astro

```diff
@@ -23,8 +23,8 @@ const getVideoThumbnail = (url: string) => {
   <Image
     class:list={['rounded-lg', 'shadow-md', 'aspect-video', 'object-cover']}
     src={getVideoThumbnail(value.url)}
-    width="auto"
-    aspectRatio={16 / 9}
+    width={240}
+    height={180}
```

Comment:
Tu dodałam:
`fit={'cover'} quality={100}`

i to poprawia jakość, ale width i height i tak musi zostać